pages/web_cloud/email_and_collaborative_solutions/zimbra/migrate_mxplan_to_zimbra/meta.yaml

```
@@ -0,0 +1,3 @@
+id: 0a46b1b1-339b-4920-bae6-f6fbdd6ee467
+full_slug: migrate-mxplan-to-zimbra
```

```suggestion
full_slug: zimbra-migrate-mxplan-to-zimbra
```
we should have a common `zimbra-`prefix to all Zimbra guides